client/components/LearnerViewDirective.js

```diff
@@ -733,6 +739,32 @@ tie.directive('learnerView', [function() {
         };
 
         /**
+          * Highlights syntax errors in the coding UI
+          *
+          * @param {number} lineNumber
+          */
+        var highlightLine = function(lineNumber) {
+          var actualLineNumber = lineNumber - 1;
+          var codeLines = document.querySelectorAll('.CodeMirror-line');
+          if ($scope.isInDarkMode) {
+            codeLines[actualLineNumber].classList.add('tie-syntax-error-line',
+              'night-mode');
```

Comment:
Ah, no, this isn't quite what I meant.

Try having your CSS rule be something like `.tie-wrapper.night-mode .CodeMirror-line.tie-syntax-error-line`. Does that work? We currently have too many "night-mode"s scattered all over the place and I'm leery of adding another one indirectly via JS.